pagespeed/system/serf_url_async_fetcher.h

```diff
@@ -74,6 +74,24 @@ struct SerfStats {
   static const char kSerfFetchFailureCount[];
   static const char kSerfFetchCertErrors[];
   static const char kSerfFetchReadCalls[];
+
+  // A fetch that finished with a 2xx or a 3xx code --- and not just a
+  // mechanically successful one that's a 4xx or such.
```

Comment:
As in HTTP reply message came through OK, so kSerfFetchFailureCount may be 0.
It's no good if all we fetch are 404s, after all.